README.md

```diff
@@ -1,4 +1,14 @@
 # DSCI_532_L02_group207_dashboards
 Milestone 1 Public Repository for Group 207
 
+#### Description 
+
+The app contains a landing page that displays the annual yield of barley for a range of varieties and sites in Minnesota for the years 1931 and 1932.  The user has an option to filter out variables from the annual yield distribution display, by the varieties of barley, by the sites, and finally by the year. There is also a map that will highlight the selected sites in Minnesota for which the yield is displayed. Users will find the app helpful to compare the yield across multiple sites in Minnesota and decide which variety has the maximum yield through the app.
```

Comment:
```suggestion
The app contains a landing page that displays the annual yield of barley for a range of varieties and sites in Minnesota for the years 1931 and 1932. The user has an option to filter out variables from the annual yield distribution display, by the varieties of barley, by the sites, and finally by the year. There is also a map that will highlight the selected sites in Minnesota for which the yield is displayed. Users will find the app helpful to compare the yield across multiple sites in Minnesota and decide which variety has the maximum yield through the app.
```

Reply:
There was an extra space after 1932 in the second line (before The user has an option...). That's all I changed.